Github-Issue-Guide.md

```diff
@@ -0,0 +1,55 @@
+# Filing an Issue on Oppia's GitHub: A Step-by-Step Guide for Non-Developers
+
+Welcome to Oppia's GitHub issue filing guide for non-developers! This step-by-step guide will help you navigate the process of creating a GitHub account and filing an issue on Oppia's GitHub repository. By following these instructions, you can contribute to improving Oppia without worrying about making mistakes. Let's get started!
+
+## Create a GitHub Account
+
+1. Visit the GitHub website at [https://github.com](https://github.com).
+
+2. Click on the "Sign up" button to create a new GitHub account.
+
+3. Follow the instructions provided by GitHub to complete the signup process.
+
+4. Verify your account by clicking on the confirmation link sent to your email.
+
+## Access Oppia's GitHub Repository
+
+1. Open your web browser and go to [Oppia's GitHub repository](https://github.com/oppia/oppia).
+
+2. You will see the main page of the Oppia repository where the development and issue tracking take place.
+
+## File a New Issue
+
+1. On the Oppia repository page, locate the "Issues" tab near the top navigation bar and click on it.
+ <img width="1437" alt="Screenshot 2023-06-08 at 23 50 59" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/1e0e3833-d04b-4507-8251-f6039a094329">
+
+2. In the "Issues" tab, you will find a green button labeled "New issue." Click on it to create a new issue.
+<img width="1440" alt="Screenshot 2023-06-08 at 23 51 58" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/05b279f5-24b9-4986-a7a9-352e882198f1">
```

Comment:
All the images seem to be in your fork. Can you add them to the oppia/oppia-web-developer-docs repo instead?

Reply:
@gp201 How can I do that ? I tried adding new assets to images folder, but i think images are not being uploaded properly even though link to image appears to be correct.

Reply:
For images ... I imagine this isn't the first time we've used images in the wiki. Maybe look at which other pages use images and how they're stored? Then you can mimic that.

Reply:
Thanks Sean, I followed Praful's [PR](https://github.com/oppia/oppia-web-developer-docs/pull/173) and made the changes.